tools/validate_spec.py

```diff
@@ -161,6 +215,112 @@ def _validate_message(
     )
 
 
+class CoverageInformation:
+    def __init__(self, packages: List[Package], coverage: bool) -> None:
+        self.__total_message_coverage: Dict[ID, Dict[Link, bool]] = {}
+        self.__spec_files: Dict[str, List[ID]] = defaultdict(lambda: [])
```

Comment:
Instead of `lambda: []` just `list` can be used.

Reply:
Done in e020204d